{"ray": {"sid": "6d8174a7e8824c8498defadc5cce8107", "uid": null, "qid": "a185a14a9fb64cb1a1f1ecd7048d7824", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [a185a14a9fb64cb1a1f1ecd7048d7824]\nTraceback (most recent call last):\n  File \"C:\\Users\\avina\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\openfabric_pysdk\\app\\worker.py\", line 90, in process\n    output = execution_callback_function(data, ray, self.state)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\avina\\Downloads\\openfabric-ai-software-engineer 2\\openfabric-ai-software-engineer\\main.py\", line 153, in execute\n    response = agent_executor.run({\"input\": text})\n               ^^^^^^^^^^^^^^^^^^\nAttributeError: 'str' object has no attribute 'run'\n", "created_at": "2023-12-31T08:19:02.619858"}], "status": "FAILED", "created_at": "2023-12-31T08:19:02.478805", "updated_at": "2023-12-31T08:19:02.619858", "finished": true}, "in": {"text": ["string"]}, "out": {}}